contracts/DiscountPhases.sol

```diff
@@ -56,6 +65,18 @@ contract DiscountPhases is StaffUtil {
 
 	constructor(IStaff _staffContract) StaffUtil(_staffContract) public {
 	}
+	
+	/**
+	function getBonus
+	
+	Internal function. Returns the amount of bonus a contributor received from a token purchase.
```

Comment:
It returns the amount of bonus a contributor should receive. This function is called during a purchase.